raylib/rlgl_purego.go

```diff
@@ -16,6 +16,7 @@ var rlLoadIdentity func()
 var rlTranslatef func(x float32, y float32, z float32)
 var rlRotatef func(angle float32, x float32, y float32, z float32)
 var rlScalef func(x float32, y float32, z float32)
+var rlMultMatrix func(m []float32)
```

Comment:
@gen2brain should this be the function signature of `C.rlMultMatrixf` or of the golang function?  If the former then it should likely be Matrix not a []float32.

I'm not familiar with this purego stuff.

Reply:
@rocktavious Yes, the C signature, is similar to `dlopen` or `libffi`. The Go functions should both be the same of course.

Reply:
@gen2brain - so just for 100% clarity - what I have right now is good - i just need to add 1 function in `rlgl_purego` that is `func MultMatrix(m Matrix) {` ?

Reply:
You need a pointer, like `var rlMultMatrix func(m *float32)`, and then pass `unsafe.Pointer(&m[0])`.

Reply:
@gen2brain - ok i think i've done this correctly now.  I'm not sure how to actually test this to see if this works.  I've never used the purego implementation.

Reply:
@rocktavious 
```console
[jupiterrider@pc42 raylib]$ GOOS=windows go test 
# github.com/gen2brain/raylib-go/raylib
./rlgl_purego.go:305:7: undefined: MatrixToFloat
./rlgl_purego.go:306:2: undefined: rlMultMatrixf
```

Reply:
@JupiterRider - the 2nd on makes sense that a typo on my part i'll fix but the first one

```
./rlgl_purego.go:305:7: undefined: MatrixToFloat
```

The function is just in the raylib-go library in https://github.com/gen2brain/raylib-go/blob/5314a4a2099ae5c4d32ad9147b5966b7e6486b5c/raylib/rcore.go#L858

So i'm not sure why its not finding that.  There is something i'm missing in my mental model of how all this pure go stuff works.

Reply:
@rocktavious 
I created a new pull request, so you can remove your purego implementation.